src/lib.rs

```diff
@@ -160,6 +160,8 @@
 //! Additional examples can be found inside of the [`examples`](https://github.com/sigstore/sigstore-rs/tree/main/examples/)
 //! directory.
 
+#![forbid(unsafe_code)]
```

Comment:
I'd suggest sticking the clippy lint configuration here as well:

```suggestion
#![forbid(unsafe_code)]
#![warn(clippy::unwrap_used, clippy::panic)]
```